product_docs/docs/biganimal/release/getting_started/creating_a_cluster/creating_an_eha_cluster.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Creating an extreme high-availability cluster
+---
+
+When you create an extreme high-availability cluster, you need to set up the data group. Extreme high-availability clusters can contain one or two data groups. 
+
+1.  Sign in to the [BigAnimal](https://portal.biganimal.com) portal.
+
+1.  Select **Create New Cluster** in the top right of the **Overview** page or **Clusters** page.
+
+1. In the **Cluster info** tab, select **Extreme High Availability**. 
+
+1. Select **Next: Cluster Settings**. 
+
+1. In the **Cluster Settings** tab, enter your cluster name and password. 
+
+1. Select **Next: Data Groups**. 
+
+1. Go to the **Nodes Settings** tab. 
+
+1. In the **Nodes** section, choose between **Two Data Nodes** and **Three Data Nodes**. The default for one data group is three data nodes. For more information on node architecture, see [Extreme High Availability](/biganimal/latest/overview/02_high_availability/#extreme-high-availability-preview).
+
+1. In the **Database Type** section:
+
+   1.  Select the type of Postgres you want to use in the **Postgres Type** field:
+
+        -   **Oracle Compatible** is powered by [EDB Postgres Advanced Server](/epas/latest/). View [a quick demonstration of Oracle compatibility on BigAnimal](../../using_cluster/06_demonstration_oracle_compatibility). EDB Postgres Advanced Server is compatible with all three cluster types.
+
+        -   **PostgreSQL Compatible** uses advanced logical replication of data and schema. It is only available if you select extreme high availability as your cluster type.
```

Comment:
Don't need to say it is only available for EHA since this topic is EHA-specific.
```suggestion
        -   **PostgreSQL Compatible** uses advanced logical replication of data and schema. 
```